nixos/modules/services/web-apps/mastodon.nix

```
@@ -425,6 +425,14 @@ in {
           Do automatic database migrations.
         '';
       };
+
+      mediaAutoRemove = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
```

```suggestion
        default = false;
```

Let's not start "automatically removing" things by default, that would probably be surprising.

---

It's more of a cleanup action, or rather a cache purge. We don't remove media from this instance. Also, it's described as a recommended setting in the mastodon admin guide. If I link to that section, maybe it's ok to leave it?

---

That makes sense, I suppose.

---

I agree with @turion, but I think it still would make sense to mention this in the release notes. Some people might have already automated this themselves and would neeed to remove that setup.

---

Actually I'd like to have more configurability here. How about 

```
services.mastodon.mediaAutoRemove.enable
services.mastodon.mediaAutoRemove.startAt (default "daily")
services.mastodon.mediaAutoRemove.olderThanDays (default 30)
```
?

This way users could choose the interval themselves according to their usecase.

---

Fine like this?

---

Yes, thank you!